### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by alissapajer**

### CODE REVIEW

File: connector/src/test/scala/quasar/qscript/Optimize.scala [hunk 1]
```diff
@@ -44,11 +55,11 @@ class QScriptOptimizeSpec extends quasar.Qspec with CompilerHelpers with QScript
   val RootI: QSI[Fix[QSI]] = DEI.inj(Const[DeadEnd, Fix[QSI]](Root))
   val UnreferencedI: QSI[Fix[QSI]] = QCI.inj(Unreferenced[Fix, Fix[QSI]]())
 
-  implicit def qsiToQscriptTotal[T[_[_]]]: Injectable.Aux[QSI, QScriptTotal[Fix, ?]] =
-    Injectable.coproduct(Injectable.inject[QScriptCore[Fix, ?], QScriptTotal[Fix, ?]],
-      Injectable.coproduct(Injectable.inject[ProjectBucket[Fix, ?], QScriptTotal[Fix, ?]],
-        Injectable.coproduct(Injectable.inject[ThetaJoin[Fix, ?], QScriptTotal[Fix, ?]],
-          Injectable.inject[Const[DeadEnd, ?], QScriptTotal[Fix, ?]])))
+  implicit def qsiToQscriptTotal[T[_[_]]]: Injectable.Aux[QSI, QST] =
```
Comment (paulp) [inline connector/src/test/scala/quasar/qscript/Optimize.scala]: The `[T[_[_]]]` isn't doing anything.

Comment (sellout) [inline connector/src/test/scala/quasar/qscript/Optimize.scala]: You should know by now that we just add `T[_[_]]` to everything, to be safe.

Comment (paulp) [inline connector/src/test/scala/quasar/qscript/Optimize.scala]: Safety first!

**Comment by sellout** [comment]
Yeah, other than what @paulp mentioned, looks good. So I guess just remove the unnecessary type parameter … which may actually fix some annoying error I get occasionally in incremental compilation?

**Comment by alissapajer** [comment]
@sellout change made

**Comment by alissapajer** [comment]
@sellout this is green

**Comment by sellout** [comment]
So … can you merge #1523 first, and we can cross our fingers that it doesn’t cause conflicts with this? I just promised @rabbitonweb that I’d try to get it in before he starts his day tomorrow.

**Pull request merged**